pkg/server/handlers.go

```diff
@@ -162,12 +162,10 @@ func (s *Server) CampaignDescribeHandler(w http.ResponseWriter, r *http.Request)
 
 // CancelHandler takes a campaignID from the user, then
 // sets its status to cancelled in the database.
-// returns the parameters of that campaign via JSON
 func (s *Server) CancelHandler(w http.ResponseWriter, r *http.Request) {
-	var q db.Query
-	var campaign db.Campaign
+	var postBody map[string]interface{}
```

Comment:
We should make a struct (could be inline) for this so we don't have to do that uint float64 dance below.

Reply:
Yup, makes total sense. Chalk another thing up to my Go n00b-ishness. Will do it this way in the future for one-off structures.